Src/Common/Domain/implementation/VolunteerOpportunity.cs

```diff
@@ -8,42 +9,60 @@ public class VolunteerOpportunity : IVolunteerOpportunity
         #region Constructors
 
         public VolunteerOpportunity()
-            : this(null)
+            : this(null, null)
         {
         }
 
-        public VolunteerOpportunity(IUser createdBy)
+        public VolunteerOpportunity(IUser createdBy, IOrganization organization)
         {
             CreatedBy = createdBy;
+            Organization = organization;
 
             RecordId = 0;
-            StartTime = DateTime.Today;
-            StopTime = DateTime.Today;
+
+            ProjectName = string.Empty;
+            ProjectDescription = string.Empty;
+
+            StartTime = DateTime.MinValue;
+            StopTime = DateTime.MinValue;
+
+            NumVolunteersNeeded = 0;

```

Comment:
Just for symmetry, what do you think about MinimumVolunteers and MaximumVolunteers?